packages/golden-layout/build/ns.js

```diff
@@ -0,0 +1 @@
+var lm={"config":{},"container":{},"controls":{},"errors":{},"items":{},"utils":{}};
```

Comment:
What's up with this file and why is it in a `build` folder? Any reason it shouldn't be in the `src` folder?

Not going to block merge for this since this isn't something we wrote, but we should clean it up unless there's a reason not to.

Reply:
It's the build starter file I guess (the build process is just concatenating several JS files)

I moved it to src and renamed it `base.js` since `ns.js` has no meaning to me